code/src/App.css

```diff
@@ -0,0 +1,12 @@
+@font-face{
+  font-family:Menlo;
+  src: url(./Menlo-Regular.ttf);
```

Comment:
Be careful with this. This font can't be rendered in Internet Explorer. A good idea is to add a `.otf` and `.eot` font as well to get it working on all browsers.